build.gradle

```diff
@@ -0,0 +1,27 @@
+buildscript {
```

Comment:
buildscript is old and deprecated,
should be using new plugins mechanism.

example one:
```
plugins {
    id 'application'
    id 'checkstyle'
    id 'io.spring.dependency-management' version '1.0.3.RELEASE'
    id 'org.springframework.boot' version '1.5.6.RELEASE'
    id 'org.owasp.dependencycheck' version '1.4.5.1'
    id 'com.github.ben-manes.versions' version '0.15.0'
    id 'org.sonarqube' version '2.5'
    id 'jacoco'
}
```

Reply:
Done. For now, just Spring-related plugins, but I'll add whatever is appropriate pull requests after this one.